sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/models/one/Extensions.java

```diff
@@ -46,6 +46,11 @@
      */
     private static final String LOC = "loc";
 
+    /**
```

Comment:
We opted to keep same JSON order as bond as One Collector team some Asimov customers are sensitive to JSON order of fields.

Device extension is located between user and os extensions in Bond.

Reply:
(Should reorder everything, the constant, the getter/setter etc... though what really matters it the order of the write method)